contracts/root/staking/StakeManagerStakingData.sol

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
+
+
+/**
+ * @title StakeManagerStakingData
+ * @notice Holds all staking related data.
+ * Note that this is contract is designed to be included in StakeManager. It is upgradeable.
+ */
+abstract contract StakeManagerStakingData {
+    uint256 internal theTotalStake;
```

Comment:
rename this to use mixedCase without articles.

Reply:
There is a name collision between the totalStake function and this variable. How should it be named so as to not have the collision?

Reply:
It's internal, so you can use `_totalStake`

Reply:
internal and private variables should use an underscore as per our naming convention